lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -39,6 +39,19 @@
       <p>${_("Click to generate a CSV file of all proctored exam results in this course.")}</p>
       <p><input type="button" name="proctored-exam-results-report" value="${_("Generate Proctored Exam Results Report")}" data-endpoint="${ section_data['list_proctored_results_url'] }"/></p>
     %endif
+
+    <p>${_("To generate a CSV file that lists all student answers to a given problem, enter the location of the problem (from its Staff Debug Info).")}</p>
+
+    <p>
+      <label>
+        <span>${_("Problem URL: ")}</span>
+        <input type="text" name="problem-url" />
```

Comment:
@sarina I'm confused - if it's not semantically correct to say *location*, why would we want to change the "url" references at all?